<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>发送数据</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="checkBox_HexSend">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Send">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Resize">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_Send">
      <item row="0" column="0">
       <widget class="QLabel" name="label_FrameHeader">
        <property name="text">
         <string>帧头   :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Data">
        <property name="text">
         <string>数据   :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_CRC">
        <property name="text">
         <string>校验码 :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_FramTail">
        <property name="text">
         <string>帧尾   :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FrameHeader">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CRC">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FrameTail">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_Data">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxReceive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>621</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>接收数据</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_Display">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>491</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>77</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_Reserve">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_Hex">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>十六进制</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_Save">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_Stop">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_Clear">
        <property name="text">
         <string>清屏</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_UartState">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>操作（&amp;F）</string>
    </property>
    <addaction name="action_SetUart"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助（&amp;H）</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_SetUart">
   <property name="text">
    <string>配置串口(&amp;S)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_FrameHeader</tabstop>
  <tabstop>textEdit_Data</tabstop>
  <tabstop>lineEdit_CRC</tabstop>
  <tabstop>lineEdit_FrameTail</tabstop>
  <tabstop>checkBox_HexSend</tabstop>
  <tabstop>pushButton_Send</tabstop>
  <tabstop>pushButton_Resize</tabstop>
  <tabstop>checkBox_Hex</tabstop>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>pushButton_Stop</tabstop>
  <tabstop>pushButton_Clear</tabstop>
  <tabstop>textBrowser_Display</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
